oss_build.sh

```diff
@@ -88,17 +88,41 @@ for python_version in $PYTHON_VERSIONS; do
     bazel clean
   fi
 
-  if [ "$python_version" = "3.6" ]; then
-    export PYTHON_BIN_PATH=/usr/bin/python3.6 && export PYTHON_LIB_PATH=/usr/local/lib/python3.6/dist-packages
-  elif [ "$python_version" = "3.7" ]; then
-    export PYTHON_BIN_PATH=/usr/local/bin/python3.7 && export PYTHON_LIB_PATH=/usr/local/lib/python3.7/dist-packages
-    ABI=cp37
-  elif [ "$python_version" = "3.8" ]; then
-    export PYTHON_BIN_PATH=/usr/bin/python3.8 && export PYTHON_LIB_PATH=/usr/local/lib/python3.8/dist-packages
-    ABI=cp38
+  if [ "$(uname)" = "Darwin" ]; then
+    if [ "$python_version" = "3.6" ]; then
+      export PYTHON_BIN_PATH=python3.6
+    elif [ "$python_version" = "3.7" ]; then
+      export PYTHON_BIN_PATH=python3.7
+      ABI=cp37
+    elif [ "$python_version" = "3.8" ]; then
+      export PYTHON_BIN_PATH=python3.8
+      ABI=cp38
+    else
+      echo "Error unknown --python. Only [3.6|3.7|3.8]"
+      exit
+    fi
+
+    export PYTHON_LIB_PATH=`$PYTHON_BIN_PATH -c 'import site; print("\\n".join(site.getsitepackages()))'`
```

Comment:
prefer use of \$() vs backticks.

Reply:
and i'm surprised you can join via a newline.  i thought space or colon were the right way?

Reply:
```
  export PYTHON_BIN_PATH=`which python${python_version}`
  export PYTHON_LIB_PATH=`${PYTHON_BIN_PATH} -c 'import site; print(site.getsitepackages()[0])'`
```